ionoscloud/data_source_dbaas_pgsql_database.go

```diff
@@ -0,0 +1,54 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services/dbaas"
+)
+
+func dataSourceDbaasPgSqlDatabase() *schema.Resource {
```

Comment:
might be useful to have a ds that returns a list of databases and even have owner as a filter

Reply:
@cristiGuranIonos both are done in cc3d6928b915b5af4bcaf43fbd60d2847e407fa5